### PR TITLE
move builder/testing framework into sdk; it imports the core but I th…

### PR DESCRIPTION
**Pull request by SwampDragons**
Move helper/builder/testing into SDK.

### CODE REVIEW

**Review by nywilken — APPROVED** [review]
Looks good to me!

File: builder/alicloud/ecs/builder_acc_test.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 
 	"github.com/aliyun/alibaba-cloud-sdk-go/services/ecs"
-	builderT "github.com/hashicorp/packer/helper/builder/testing"
+	builderT "github.com/hashicorp/packer/packer-plugin-sdk/builder/testing"
```
Comment (nywilken) [inline builder/alicloud/ecs/builder_acc_test.go:11]: Once broken out into a separate ropo, what do you think about renaming this to `acctest` or creating a new pkg that allows for the acceptance testing of builds including provisioners and post-processors?

**Review by azr — APPROVED** [review]

**Review by sylviamoss — APPROVED** [review]
LGTM! 👍

File: builder/alicloud/ecs/builder_acc_test.go (hunk 1 above)
Comment (SwampDragons) [reply builder/alicloud/ecs/builder_acc_test.go:11]: yeah I like that idea; changing the name to acctest would make it a lot clearer architecturally what's going on. I'll do that in a final PR once I've finished cleaning up the package and its usage.

**Pull request merged**